src/frequenz/channels/util/_file_watcher.py

```diff
@@ -14,7 +17,17 @@
 from .._exceptions import ReceiverStoppedError
 
 
-class FileWatcher(Receiver[pathlib.Path]):
+@dataclass(frozen=True)
+class FileEvent:
+    """A class to represent a file event useful for notifying file changes."""
+
+    type: Change
```

Comment:
This still needs to be `FileWatcher.EventType`, not `Change`. We shouldn't expose anything from `watchfiles` to the user.

Reply:
yeah my bad, will update it